task/sast-shell-check/0.1/sast-shell-check.yaml

```diff
@@ -0,0 +1,175 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "konflux"
+  name: sast-shell-check
+spec:
+  description: >-
+    ShellCheck is a static analysis tool, gives warnings and suggestions for bash/sh shell scripts.
+  results:
+    - description: Tekton task test output.
+      name: TEST_OUTPUT
+  params:
+    - name: image-url
+      description: Image URL.
+      type: string
+      default: ""
+    - name: image-digest
+      description: Image digest to report findings for.
+      type: string
+      default: ""
+    - name: KFP_GIT_URL
+      description: git repository to download known false positives files from
+      type: string
+      default: ""
+    - name: PROJECT_NVR
+      description: Name-Version-Release (NVR) of the scanned project, used to find path exclusions
+      type: string
+      default: ""
+    - name: RECORD_EXCLUDED
+      type: string
+      description: |
+        Whether to record the excluded findings (default to false).
+        If `true`, the excluded findings will be stored in `excluded-findings.json`.
+      default: "false"
+    - name: CSGREP_EVENT_FILTER
+      description: ShellCheck event filter for csgrep.
+      type: string
+      default: '\[SC(1020|1035|1054|1066|1068|1073|1080|1083|1099|1113|1115|1127|1128|1143|2043|2050|2055|2057|2066|2069|2071|2077|2078|2091|2092|2157|2171|2193|2194|2195|2215|2216|2218|2224|2225|2242|2256|2258|2261)\]$'
+  steps:
+    - name: sast-shell-check
+      image: quay.io/redhat-appstudio/konflux-test:v1.4.5@sha256:801a105ba0f9c7f58f5ba5cde1a3b4404009fbebb1028779ca2c5de211e94940
+      # per https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting
+      # the cluster will set imagePullPolicy to IfNotPresent
+      workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)
+      env:
+        - name: KFP_GIT_URL
+          value: $(params.KFP_GIT_URL)
+        - name: PROJECT_NVR
+          value: $(params.PROJECT_NVR)
+        - name: RECORD_EXCLUDED
+          value: $(params.RECORD_EXCLUDED)
+        - name: CSGREP_EVENT_FILTER
+          value: $(params.CSGREP_EVENT_FILTER)
+      script: |
+        #!/usr/bin/env bash
+        set -x
+        . /utils.sh
+
+        # install dependencies
+        microdnf install -y csdiff
```

Comment:
This command can be removed if you update the reference to the `konflux-test` image, thanks to https://github.com/konflux-ci/konflux-test/pull/292

Reply:
addressed in https://github.com/konflux-ci/build-definitions/pull/1437/commits/a09977ba917bdd71155cce6bf670e76a40a23ba6